banyand/measure/measure_write.go

```diff
@@ -60,7 +61,7 @@ func (s *measure) write(md *commonv1.Metadata, shardID common.ShardID, entity []
 	if err != nil {
 		return err
 	}
-	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	ctx, cancel := context.WithTimeout(context.TODO(), 5*time.Second)
```

Comment:
Why do you change it to `TODO`? This context is the top-level, `Background` is better here.

Reply:
There is no difference between `TODO` and `Background` from the implementation. Anyway, reverted.